<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDialogWidget</class>
 <widget class="QWidget" name="EventDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Name</string>
     </property>
     <property name="buddy">
      <cstring>ed_name</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="ed_name"/>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="ed_stageCount">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Stage count</string>
     </property>
     <property name="buddy">
      <cstring>ed_stageCount</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Date</string>
     </property>
     <property name="buddy">
      <cstring>ed_date</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Place</string>
     </property>
     <property name="buddy">
      <cstring>ed_place</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Des&amp;cription</string>
     </property>
     <property name="buddy">
      <cstring>ed_description</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Main r&amp;eferee</string>
     </property>
     <property name="buddy">
      <cstring>ed_mainReferee</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>D&amp;irector</string>
     </property>
     <property name="buddy">
      <cstring>ed_director</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="ed_date"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="ed_place"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="ed_description"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="ed_mainReferee"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="ed_director"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Event ID</string>
     </property>
     <property name="buddy">
      <cstring>ed_name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ed_eventNameId"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
